skyfield/starlib.py

```diff
@@ -106,6 +106,18 @@ def from_dataframe(cls, df):
             epoch=epoch,
         )
 
+    @classmethod
```

Comment:
Let's avoid adding any new methods to `Star`, since it will become a very large class if it has to learn details about every kind of external catalog. Instead, it is designed to stay simple, and every catalog simply needs to format its Pandas dataframe for the `from_dataframe()` method, and then the star-catalog details remain over in the code specific to those catalogs, instead of needing updates to the class itself, which ideally shouldn't even need to know that different catalogs exist.